pyproject.toml

```diff
@@ -1,3 +1,88 @@
 [build-system]
-requires = ["setuptools>=41.2", "setuptools_scm", "wheel"]
-build-backend = "setuptools.build_meta"
+requires = ["hatchling", "hatch-requirements-txt", "hatch-vcs"]
+build-backend = "hatchling.build"
+
+[project]
+name = "conda_lock"
+description = "Lockfiles for conda"
+authors = [
+    { "name" = "Marius van Niekerk", email = "marius.v.niekerk@gmail.com" },
+]
+readme = "README.md"
+license = "MIT"
+requires-python = ">=3.6"
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Science/Research",
+    "Operating System :: OS Independent",
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+]
+dynamic = ["dependencies", "version"]
+license-files = { paths = ["LICENSE"] }
+
+[project.optional-dependencies]
+pip_support = ["poetry ==1.1.*"]
+
+[project.scripts]
+conda-lock = "conda_lock:main"
+
+[project.urls]
+Homepage = "https://github.com/conda-incubator/conda-lock"
+Repository = "https://github.com/conda-incubator/conda-lock"
+Documentation = "https://conda-incubator.github.io/conda-lock/"
+"Issue Tracker" = "https://github.com/conda-incubator/conda-lock/issues"
+PyPI = "https://pypi.org/project/conda-lock/"
+"Conda-Forge Feedstock" = "https://github.com/conda-forge/conda-lock-feedstock"
+"Anaconda.org" = "https://anaconda.org/conda-forge/conda-lock"
+
+[tool]
+
+[tool.check-manifest]
+ignore = [
+    "*.yml",
+    ".coveragerc",
+    "Makefile",
+    "docs",
+    "docs/*",
+    "*.enc",
+    "tests",
+    "tests/*",
+]
+
+[tool.coverage.run]
+omit = "conda_lock/vendor/*"
```

Comment:
seems these should be a lists?

Reply:
Ah yes, I meant to change it, but I was directly translating at first... commit pending...

Reply:
Done